test/src/tests-persistence.ts

```diff
@@ -1695,12 +1729,13 @@ describe('migrations and persistence-related', function() {
         });
 
         setCookie(workspaceCookieName, cookies);
-        mParticle.useCookieStorage = true;
+        // FIXME: Should this be in configs or global?
```

Comment:
I think perhaps it should be in a before clause for a separate section.  The persistence tests definitely need to be organized better with more `describe` blocks.  The lack of them right now makes things confusing as to when one portion of related tests ends and another begins - this is an issue overall with how tests are set up and not just persistence.  Something we should 100% do better in jest.